511.xml

```diff
@@ -0,0 +1,89 @@
+FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+The above license is used as a license under copyright only.  Please
+reference the OMA IPR Policy for patent licensing terms:
+https://www.omaspecworks.org/about/intellectual-property-rights/
+
+-->
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M-v1_1.xsd">
+    <Object ObjectType="MODefinition">
+        <Name>Compact Composite</Name>
+        <Description1>This Object enables collecting a set of Resource values from other Objects into a single resource representation.</Description1>
+        <Description2/>
+        <ObjectID>511</ObjectID>
+        <ObjectURN>urn:oma:lwm2m:oma:511</ObjectURN>
+        <LWM2MVersion>1.1</LWM2MVersion>
+        <ObjectVersion>1.0</ObjectVersion>
+        <MultipleInstances>Multiple</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Resources>
+            <Item ID="0">
+                <Name>Target Links</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Mandatory</Mandatory>
+                <Type>Corelnk</Type>
+                <RangeEnumeration/>
+                <Units/>
+                <Description>One or more CoRE Links to Readable Single Resources and/or Resource Instances of Readable Single Resources in the same LwM2M Client, from which the values will be stored in the Compact Data array. Each link corresponds to an item in the Compact Data array: first link points to the resource instance with the value for the first array item, second link to the instance with the second array item, etc.</Description>
+            </Item>
+            <Item ID="1">
+                <Name>Compact Data</Name>
+                <Operations>R</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Mandatory</Mandatory>
+                <Type>Opaque</Type>
+                <RangeEnumeration/>
+                <Units/>
+                <Description>CBOR array of resource values in the same order as the links in the Target Links resource. CBOR simple value of undefined should be used as the array item if an error occurs while reading the value for the corresponding resource (e.g., link pointing to a resource that doesn't exist or the resource is not allowed to be read because of access control rules).</Description>
```

Comment:
```suggestion
                <Description>CBOR array of Resource or Resource Instance values in the same order as the links in the Target Links Resource. CBOR simple value of undefined should be used as the array item if an error occurs while reading the value for the corresponding Resource or Resource Instance (e.g., link pointing to a Resource that doesn't exist or the Resource is not allowed to be read because of access control rules).</Description>
```